src/ansys_sphinx_theme/theme/ansys_sphinx_theme/layout.html

```diff
@@ -6,6 +6,7 @@
 
   <!-- Include desired assets -->
   <script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
+  <script src="https://cdn.plot.ly/plotly-3.0.1.min.js"></script>
```

Comment:
Missing Plotly.js: Plotly was not working in our case because plotly.js was missing. I have now added it, and it is working correctly in sphinx-gallery scripts.

Attaching images from latest build artifacts
![Screenshot 2025-03-14 at 17 18 33](https://github.com/user-attachments/assets/c3d8e8d1-6a4a-4430-82db-1d89ac7fe340)

Reply:
I am very surprised we need to add this. As far as I am concerned, this only affects out project, right? If so, let's add this in the `conf.py`. Otherwise, we are distributing `plotly.js` across all out projects.